sdk/Sdk/Tasks/ZipDeploy/StringMessages.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+﻿namespace Microsoft.NET.Sdk.Functions.Tasks
+{
+    public static class StringMessages
```

Comment:
vs-build-sdk uses Resources.resx for messages -- https://github.com/Azure/azure-functions-vs-build-sdk/blob/main/src/Microsoft.NET.Sdk.Functions.MSBuild/Properties/Resources.resx

But I just put them here for simplicity.